.github/workflows/on-tag.yml

```diff
@@ -0,0 +1,89 @@
+name: On Tag - Release
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+      - 'v*.*.*-*'
+
+jobs:
+  debug-print:
+    name: <Debug> Branch/tag - ${{github.ref}}
+    steps:
+      - name: step1
+        id: step1
+        run: |
+          echo "no-op"
+
+  build-psibase:
+    name: Run Builds
+    uses: ./.github/workflows/ubuntu-builder.yml
+
+  prerelease-check:
+    name: Prerelease Check
+    runs-on: ubuntu-latest
+    outputs:
+      prerelease: ${{ steps.set-prerelease-var.outputs.prerelease }}
+    steps:
+      - name: Set prerelease variable
+        id: set-prerelease-var
+        shell: bash
+        run: |
+          if [[ ${{ github.ref_name }} == *-* ]]; then
+              echo prerelease=true >>$GITHUB_OUTPUT
+          else
+              echo prerelease=false >>$GITHUB_OUTPUT
+          fi
+
+  create-psibase-release:
+    name: Create Psibase Release
+    needs:
+      - build-psibase
+      - prerelease-check
+    steps:
+      - name: "Build Changelog"
+        id: build_changelog
+        uses: mikepenz/release-changelog-builder-action@v4
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Download build artifacts
+        uses: actions/download-artifact@v3
+
+      - name: <Debug> Display structure of downloaded files
+        run: ls -R
+
+      - name: Create new Github Release
+        id: create_release
+        uses: softprops/action-gh-release@v1
+        with:
+          files: |
+            psidk-ubuntu-2004.tar.gz
+            psidk-book-2004.tar.gz
```

Comment:
Removed 20.04

Reply:
In the ubuntu-builder, I dropped the version from the filename, which means the 20.04 artifact will be overwritten when 22.04 builds the book, but I guess that's fine since they aren't distro specific.